api/auth/signer/v4/chunked_reader.go

```diff
@@ -199,7 +202,7 @@ func readChunkLine(b *bufio.Reader) ([]byte, []byte, error) {
 		}
 		return nil, nil, err
 	}
-	if len(p) >= maxLineLength {
+	if len(p) >= MaxChunkedPayloadLineLength {
```

Comment:
And here it's too late, panic happens earlier than that. The real question is why `ReadSlice` doesn't return `ErrBufferFull` above.

Reply:
Moving these rows upper is not a solution either, because panic inside `bufio.ReadSlice` and we just don't have the opportunity to check the length

Reply:
I know, but we also need to know _why_ it panics. It shouldn't.